server/src/projects/project-applicants/project-applicants.attrs.ts

```diff
@@ -0,0 +1,6 @@
+export const PROJECTAPPLICANT_ATTRS = [
+  'dcp_name',
+  'emailaddress',
+  'dcp_projectapplicantid',
+  'dcp_applicantrole',
+];
```

Comment:
nit: line break here

Reply:
We should all set up our text editors to trim trailing whitespace and ensure one trailing new line upon save. Atom's "whitespace" package does this and I often get lots of whitespace diffs when making small changes to files others have worked on.